src/public/app/player/player.component.ts

```diff
@@ -1,46 +1,73 @@
-import { Component, Input, OnChanges, SimpleChange, Output, EventEmitter } from 'angular2/core';
-import { ScPlayer } from '../soundcloud/soundcloud';
-import SoundcloudConnect from '../soundcloud/soundcloud.service';
-import { QueuedUpSong, ScPlayerEvent } from '../core/types';
+import {
+    Component,
+    Input,
+    OnChanges,
+    SimpleChange,
+    Output,
+    EventEmitter,
+    ChangeDetectorRef,
+} from 'angular2/core';
+    import SoundcloudConnect from '../soundcloud/soundcloud.service';
+import { ScPlayer, QueuedUpSong, ScPlayerEvent } from '../core/types';
 
 const template: string = `
-<div class="row">
-    <div class="col s6">
-        {{queuedSong ? queuedSong.song.title : "None selected"}}
+<div class="player-container">
+    <div class="row">
+        <div class="col s11">
+            {{ queuedSong ? queuedSong.song.title : "No song selected" }}
+        </div>
+        <div class="col s1">
+            {{ seekTime || '--' }}
+        </div>
     </div>
-    <div class="col s6">
-         <i (click)="stop()"
-            class="small material-icons pointer">stop</i>
-
-        <i (click)="play()"
-            class="small material-icons pointer">play_arrow</i>
-
+    <div class="row">
+        <div class="col s12">
+            <a (click)="play()" class="btn-floating  waves-effect waves-light teal accent-2">
+                <i class="material-icons">play_arrow</i>
+            </a>
+             <a (click)="stop()" class="btn-floating  waves-effect waves-light teal accent-2">
+                <i class="material-icons">stop</i>
+             </a>
+              <a (click)="next()" class="btn-floating  waves-effect waves-light teal accent-2">
+                <i class="material-icons">skip_next</i>
+               </a>
+        </div>
+        
     </div>
 </div>
 `;
 
 @Component({
     template,
     selector: 'player',
+    styles: [`
+        .player-container {
+            margin: 10px;
+        }
+    `, ] ,
 })
 class Player implements OnChanges {
 
     @Input() public queuedSong: QueuedUpSong;
-    @Output() public songFinished: EventEmitter = new EventEmitter();
+    @Output() public songFinished: EventEmitter<String> = new EventEmitter<String>();
```

Comment:
ah thats a typo. Im not sure but should be `string`

On Tue, Apr 19, 2016 at 5:57 PM, dswen <notifications@github.com> wrote:

> In src/public/app/player/player.component.ts
> <https://github.com/ryanquinn3/beetz/pull/8#discussion_r60337004>:
>
> >  })
> >  class Player implements OnChanges {
> >
> >      @Input() public queuedSong: QueuedUpSong;
> > -    @Output() public songFinished: EventEmitter = new EventEmitter();
> > +    @Output() public songFinished: EventEmitter<String> = new EventEmitter<String>();
>
> what's the difference between string and String?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ryanquinn3/beetz/pull/8/files/2aabd0b9fad287cfc337421e70eb4c4f70cfdaed#r60337004>
>